PowerEditor/src/ScitillaComponent/FindReplaceDlg.cpp

```diff
@@ -32,6 +32,7 @@
 #include "Notepad_plus_msgs.h"
 #include "UniConversion.h"
 #include "localization.h"
+#include "../Utf8.h"

```

Comment:
The `../` part not needed here, I think.

Reply:
I copied it from Common.cpp, of course, without thinking about it. Removing the `../` from the FindReplaceDlg.cpp ...